### PR TITLE
Adding current Cordova CLI to the kit listing

### PR DESCRIPTION
**Pull request by joshuaskelton**
taco_kit_list_cli_anim

### CODE REVIEW

File: src/taco-cli/cli/kit.ts [hunk 1]
```diff
@@ -158,10 +163,19 @@ class Kit extends commands.TacoCommandBase {
         var availableKits: INameDescription[] = [];
         var currentKitId: string = "";
 
-        return Kit.getCurrentKitInfo().then(function (kitId: string): Q.Promise<any> {
-            currentKitId = kitId;
-            if (kitId) {
-                logger.log(resources.getString("CommandKitListCurrentKit", kitId));
+        return Kit.getCurrentKitInfo().then(function (kitInfo: IKitInfo): Q.Promise<any> {
+            currentKitId = kitInfo.kitId;
+            var logLine = false;
```
Comment (MSLaguana) [inline src/taco-cli/cli/kit.ts]: `var logLine: boolean = false; ` so we are ready for Nisheet's PR to be merged

File: src/taco-cli/cli/kit.ts [hunk 2]
```diff
@@ -368,10 +382,10 @@ class Kit extends commands.TacoCommandBase {
     /**
      * Pretty prints the current Kit/Cordova CLI info
      */
-    private static getCurrentKitInfo(): Q.Promise<string> {
-        var deferred = Q.defer<string>();
-        return projectHelper.getProjectInfo().then(function (projectInfo: projectHelper.IProjectInfo): Q.Promise<string> {
-            deferred.resolve(projectInfo.tacoKitId);
+    private static getCurrentKitInfo(): Q.Promise<IKitInfo> {
+        var deferred = Q.defer<IKitInfo>();
```
Comment (MSLaguana) [inline src/taco-cli/cli/kit.ts]: `var deferred: Q.Deferred<IKitInfo> = `

**Comment by MSLaguana** [comment]
Looks good

**Comment by meenakb** [comment]
:+1:

**Comment by meenakb** [comment]
and :+1:  :+1:  for the gif :)

**Comment by nisheetjain** [comment]
:open_mouth:  for the gif

**Pull request merged**